parser/parse.go

```diff
@@ -749,3 +753,14 @@ func (b *Builder) addVariable(u types.Universe, useName *types.Name, in *tc.Var)
 	out.Underlying = b.walkType(u, nil, in.Type())
 	return out
 }
+
+// importPathToFullPackage takes an import path and returns the actual package
+// it doesn't work for any folder actually called `vendor` (we don't dep any today),
+// so hopefully we keep up the streak
+func importPathToFullPackage(importPath string) string {
```

Comment:
sorry to ask for a change but can we call this "importPathToCanonical" or "canonicalizeImportPath" or something?